spirv_msl.hpp

```diff
@@ -27,14 +27,8 @@
 namespace spirv_cross
 {
 
-// Options for compiling to Metal Shading Language
-struct MSLConfiguration
-{
-	bool flip_vert_y = false;
-	bool flip_frag_y = false;
-	bool is_rendering_points = false;
-	std::string entry_point_name;
-};
+// Deprecated legacy syntax
+#define MSLConfiguration CompilerMSL::Options
```

Comment:
Can this be a typedef instead ala using MSLConfiguration = CompilerMSL::Options?